test/test_algorithm.py

```diff
@@ -26,13 +26,15 @@ class AlgorithmTest(TestCase):
         'music_copy'
     )
 
-    def setUp(self):
+    @classmethod
```

Comment:
Why do we have `@classmethod` decorator if the `cls` attribute is not used anyway?

I guess it would be useful to replace `AlgorithmTest` by `cls` in case of inheritance to take advantage of polymorphism, but since it is not the case.. am I missing something?

Reply:
It is needed in order to simply override the inherited method from `TestCase`. Not much more to it.